lib/exq.ex

```diff
@@ -24,6 +24,7 @@ defmodule Exq do
       worker(Redix, [redis_opts, connection_opts]),
       worker(Exq.Stats.Server, [opts]),
       worker(Exq.Enqueuer.Server, [opts]),
+      worker(Exq.Middlewares.Server, [opts]),
       supervisor(Exq.Worker.Supervisor, [opts]),
       worker(Exq.Manager.Server, [opts]),
```

Comment:
@zhongwencool I just noticed this - should the manager be higher up the chain?  Seems like it should be stopped only after the workers done.